build/purger/purger.py

```diff
@@ -273,7 +280,9 @@ def _update_user_activity_for_network(
     ]
 
     due_users = list()
-
+    click.secho(
+        f"Presences of {possible_candidates} users will be fetched due to possible inactivity. This might take a while."
```

Comment:
Nitpick: AFAIU `possible_candidates` could be a very long list. Maybe it would make the message more readable if the list is printed at the end.

Reply:
of course, it should be called `len(possible_candidates`